### ISSUE TITLE
[Generic attributes]

### ISSUE THREAD

**Issue by ly3027929699** [DaZombieKiller/UnityRoslynUpdater#2]
thanks for bringing c#12 to my unity poject.
I can use the [Generic attributes] on class unless struct. The class with  [Generic attributes] would not crash.

**Comment by DaZombieKiller** [comment]
Yeah, there are a few cases where using generic attributes *might* not crash, I listed it as "Crash" to warn users that it's not 100% guaranteed to work.

If you have an `[assembly:]` or `[module:]` targeted generic attribute, that will also crash.

There's also no support for using open generic types when calling `GetCustomAttributes`, but that's to be expected since it requires runtime changes.
